### ISSUE TITLE
Cannot invoke method toInteger() on null object

### ISSUE THREAD

**Issue by rootbear75** [LBBO/netflix-migrate#38]
Following instructions to import viewing history into my gf's netflix account since we're merging accounts.

Got this error:

```
Successfully switched profile!
Reading data from ratings.json
Found rating history and viewing history
Setting rating for Bill Burr: Paper Tiger: thumbs down
Error: Cannot invoke method toInteger() on null object
    at Request._callback (C:\Users\Rootbear\AppData\Roaming\npm\node_modules\netflix-migrate\node_modules\netflix2\lib\netflix2.js:302:23)
    at Request.self.callback (C:\Users\Rootbear\AppData\Roaming\npm\node_modules\netflix-migrate\node_modules\request\request.js:185:22)
    at Request.emit (events.js:198:13)
    at Request.<anonymous> (C:\Users\Rootbear\AppData\Roaming\npm\node_modules\netflix-migrate\node_modules\request\request.js:1161:10)
    at Request.emit (events.js:198:13)
    at IncomingMessage.<anonymous> (C:\Users\Rootbear\AppData\Roaming\npm\node_modules\netflix-migrate\node_modules\request\request.js:1083:12)
    at Object.onceWrapper (events.js:286:20)
    at IncomingMessage.emit (events.js:203:15)
    at endReadableNT (_stream_readable.js:1145:12)
    at process._tickCallback (internal/process/next_tick.js:63:19)
Error: Could not set thumb rating for Bill Burr: Paper Tiger. For more information, please see previous log statements.
    at C:\Users\Rootbear\AppData\Roaming\npm\node_modules\netflix-migrate\main.js:269:10
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Comment by LBBO** [comment]
Hey @rootbear75,

thanks for reporting this issue! I can confirm that the error is thrown as a result of Netflix answering with a 500 Internal Error code when a rating is supposed to be set. Probably, they changed their API again; I'll look into that right away.

By the way: this project is not able to transfer viewing history yet. It is exported (so you will always have your old data in the JSON file) but it can't be uploaded yet, I'm afraid.

**Comment by LBBO** [comment]
Update: I found and fixed the issue. The update should be published on npm within a few minutes.

Technical details for documentation purposes: the rating data used to be sent to Netflix as a query string (such as `https://www.netflix.com/someUrl?key=value&foo=bar`) and this did not set the `content-type: application/json` header. Netflix didn't use to have a problem with that, but apparently it now causes a 500 Internal Error now. The solution was to pass the data in the requests body (the method must be set to `POST` to transmit a body!) because then the option `json: true` (which was already set) sets said header.

Probably, some other functions have the same problem, but I can't explore that theory today.

So, TL;DR: the `content-type: application/json` header should have been set when uploading ratings but never was and Netflix suddenly started caring.

**Comment by rootbear75** [comment]
no worries. Ratings are mostly what i need so at least netflix knows kind of what I watched beforehand. Thanks for the quick response tho!!
